components/ProjectModal/index.tsx

```diff
@@ -73,6 +85,85 @@ export default function ProjectModal({
           </div>
         </div>
         <div>{size}</div>
+        <AllKDMS>
+          {key_development_milestones?.map(
+            (milestone: Milestone, i: number) => {
+              let milestoneLabel = '';
+              let icon = <></>;
+              let kdmNumber = 0;
+
+              switch (milestone.milestoneTitle) {
+                case 'Entry to NYISO Queue':
+                  milestoneLabel = 'NYISO queue entered';
+                  kdmNumber = 1;
+                  break;
+                case 'Application for permit to ORES':
+                  milestoneLabel = 'ORES permit applied';
+                  kdmNumber = 2;
+                  break;
+                case 'Issuance of permit from ORES':
+                  milestoneLabel = 'ORES permit issued';
+                  kdmNumber = 3;
+                  break;
+                case 'Winning a contract award from NYSERDA':
+                  milestoneLabel = 'NYSERDA contracted';
+                  kdmNumber = 4;
+                  break;
+                case 'Execution of an Interconnection Agreement (IA)':
+                  milestoneLabel = 'IA executed';
+                  kdmNumber = 5;
+                  break;
+                case 'Start of operations':
+                  milestoneLabel = 'Operations begun';
+                  kdmNumber = 6;
```

Comment:
May be useful to store milestone titles and labels in some sort of enum structure in case their underlying value changes in the future.